libs/blocks/bulk/bulk.js

```diff
@@ -67,117 +67,117 @@ function prettyDate() {
-                          <div class="bulk-status-footer-preview-complete">job complete</div>
-                          <div class="bulk-status-footer-preview-date">${timeStamp}</div>
-                          <div class="bulk-status-footer-preview-success">successful: ${completion.preview.success} / ${completion.preview.total}</div>
-                      `}
-                  </div>
-              </td>
-              <td></td>
-              <td>
-                  <div class="bulk-status-footer-publish">
-                      ${completion.publish.total > 0 && html`
-                          <div class="bulk-status-footer-publish-complete">job complete</div>
-                          <div class="bulk-status-footer-publish-date">${timeStamp}</div>
-                          <div class="bulk-status-footer-publish-success">successful: ${completion.publish.success} / ${completion.publish.total}</div>
-                      `}
-                  </div>
-              </td>
-              <td></td>
-          </tr>
-      </table>
+    <div class="bulk-job">
+      <div class="bulk-job-status">
+        ${completion.preview.total > 0 && html`
+          <ul class="bulk-job-preview">
+            <li><span>Preview Job Complete:</span> ${timeStamp}</li>
+            <li><span>Successful:</span> ${completion.preview.success} / ${completion.preview.total}</li>
+          </ul>
+        `}
+        ${completion.publish.total > 0 && html`
+        <ul class="bulk-job-publish">
+          <li><span>Publish Job Complete:</span> ${timeStamp}</li>
+          <li><span>Successful:</span> ${completion.publish.success} / ${completion.publish.total}</li>
+        </ul>
+        `}
+      </div>
+    </div>
   `;
 }
 
-function Status({
-  valid, urlNumber, bulkTriggered, submittedAction, result, resultsElt,
-  completion, copyResults, showHideUrls, bulkInputToggleElt,
-}) {
+function Status({ valid, urlNumber, bulkTriggered, submittedAction, result, resultsElt, completion }) {
   return valid && html`
     <div class="bulk-status">
-        <div class="bulk-status-title-container">
-            <${StatusTitle}
-                bulkTriggered=${bulkTriggered}
-                submittedAction=${submittedAction}
-                copyResults=${copyResults}
-                showHideUrls=${showHideUrls}
-                bulkInputToggleElt=${bulkInputToggleElt}
-                urlNumber=${urlNumber} />
-        </div>
-        <div class="bulk-status-content-container">
-            <${StatusContent}
-                resultsElt=${resultsElt}
-                result=${result} />
-        </div>
-        <div class="bulk-status-footer-container">
-            <${StatusFooter}
-                completion=${completion} />
-        </div>
+      <div class="bulk-status-info">
+        <${StatusTitle}
+          bulkTriggered=${bulkTriggered}
+          submittedAction=${submittedAction}
+          urlNumber=${urlNumber} />
+        <${BulkStatus}
```

Comment:
To be in sync with `StatusTitle` and `StatusContent`, I'd rename it: `StatusCompletion`